### DISCUSSION TITLE
Opening a Pluto Notebook with binder

### DISCUSSION THREAD

**Discussion by VarLad** [fonsp/Pluto.jl#1803] [Questions about using Pluto]
Is there a link at the end of which I can paste the link to my Pluto notebook (stored on Github) and that notebook will open with Binder?

Something like, https://www.something.com/something/something/url=https://raw.gith.....

(sorry if the question sounds super specific)

If not, then is there some other way easily get a binder link to the Pluto Notebook I've stored?

**Comment by icweaver** [comment]
Sure!

http://pluto-on-binder.glitch.me/
